react/week3/timeregistration/src/EmployeeSearch.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from 'react'
+import TimeCalculator from './TimeCalculator';
+import { registrationDetails } from './registrationDetails';
+
+
+
+function EmployeeSearch() {
+    
+    const [search, setSearch] = useState([]);
+    return (
+        <div>
+          <input
+              type="search"
+              placeholder="Search by Name"
+              onChange={(e) => {
+                if (search.length !== 0) {
```

Comment:
Just a suggestion here , better take this logic out in separate function and call the function name here.